packages/angular/.eslintrc.json

```diff
@@ -33,6 +33,68 @@
       "rules": {
         "@nx/nx-plugin-checks": "error"
       }
+    },
+    {
+      "files": ["./package.json"],
+      "parser": "jsonc-eslint-parser",
+      "rules": {
+        "@nx/dependency-checks": [
+          "error",
+          {
+            "ignoredDependencies": [
+              "nx",
```

Comment:
Hm.. `nx` should not be ignored.

Reply:
`nx` is assumed to be installed or else the plugin cannot work.